elpis/engines/hft/objects/transcription.py

```diff
@@ -48,50 +48,38 @@ def __init__(self, **kwargs) -> None:
         self.build_stage_status(stage_names)
 
     def transcribe(self, on_complete: callable = None) -> None:
-        logger.info("==== Load processor and model ====")
+        self._set_stage(LOAD_MODEL)
+        logger.info("==== Load model ====")
         self._set_finished_transcription(False)
         processor, model = self._get_wav2vec2_requirements()
+        self._set_stage(LOAD_MODEL, complete=True)
 
-        # Load audio
         self._set_stage(LOAD_AUDIO)
         logger.info("=== Load audio")
         audio_input, _ = resampler.load_audio(
             self.audio_file_path, target_sample_rate=HFTTranscription.SAMPLING_RATE
         )
         self._set_stage(LOAD_AUDIO, complete=True)
 
-        # Pad input values and return pt tensor
-        self._set_stage(PROCESS_INPUT)
-        logger.info("==== Process input ====")
-        input_values = processor(
-            audio_input, sampling_rate=HFTTranscription.SAMPLING_RATE, return_tensors="pt"
-        ).input_values
-        self._set_stage(PROCESS_INPUT, msg="Processed input values")
-
-        # Retrieve logits & take argmax
-        with torch.no_grad():
-            logits = model(input_values).logits
-        predicted_ids = torch.argmax(logits, dim=-1)
-        self._set_stage(PROCESS_INPUT, complete=True)
-
-        # Transcribe
         self._set_stage(TRANSCRIPTION)
-        transcription = processor.decode(predicted_ids[0])
-        logger.info(f"{transcription=}")
+        logger.info("=== Inference pipeline")
```

Comment:
```suggestion
        logger.info("=== Inference pipeline ===")
```

Reply:
changed this and others that were ==== to use ===